pipelines/pipeline_navvis_rig.py

```diff
@@ -302,6 +304,13 @@ def run(
         default=False,
         help="Write out QR maps in json format. Default: False.",
     )
+    parser.add_argument(
+        "--downsample_max_edge",
+        type=int,
+        default=None,
```

Comment:
I guess its ok to not have a default valye and it is checked later for None?

Reply:
Yes, it is the default value in the `run_navvis_to_capture` script, if `downsample_max_edge` is `None` it will take the image/tile size:
```
    if downsample_max_edge is None:
        camera_params = ('SIMPLE_PINHOLE', tiles.width, tiles.height, fx, cx, cy)
```
https://github.com/microsoft/lamar-benchmark/blob/be0f26932f02d794e9412de4ac4b72c744be70a1/scantools/run_navvis_to_capture.py#L75-L76